code/lib/cli/src/upgrade.ts

```diff
@@ -135,6 +137,19 @@ export const addExtraFlags = (
   );
 };
 
+const addNxPackagesToReject = (flags: string[]) => {
+  const newFlags = [...flags];
+  if (flags.indexOf('--reject') > -1) {
+    newFlags[
+      flags.indexOf('--reject') + 1
+    ] = `/(preset-create-react-app|@nrwl/storybook|@nx/storybook)/`;
```

Comment:
I could not find an optimal way to make this actually scalable (like if someone adds more `--reject`) things. If it were to be made scalable, then we would need to agree that it's always going to be a pattern/regex, which ends in `/`, and then I could add the pattern for the Nx packages before the `/` if it ends with `/`. Because it's not effective if you combine regex with string. Then it treats all as string, and it only ignores the `@nrwl/storybook,@nx/storybook`.

Reply:
I added the "logic" of identifying if it starts and ends with slash..